scrapers/AdultEmpireCash.yml

```diff
@@ -0,0 +1,126 @@
+name: AdultEmpireCash
+# Sites ("Brands") from https://www.adultempirecash.com
+sceneByURL:
+  - action: scrapeXPath
+    scraper: sceneScraper
+    url:
+      - concoxxxion.com/
+      - elegantangel.com/
+      - filthykings.com/
+      - filthypov.com/
+      - forbiddenfruitsfilms.com/
+      - hornyhousehold.com/
+      - jayspov.net/
+      - jodiwest.com/
+      - kingsoffetish.com/
+      - latinoguysporn.com/
+      - lethalhardcore.com/
+      - manipulativemedia.com/
+      - mypervyfamily.com/
+      - pornstarstroker.com/ # aggregator site?
+      - reaganfoxx.com/
+      - severesexfilms.com/
+      - thirdworldxxx.com/
+      - touchmywife.com/
+      - westcoastproductions.com/
+
+  # VR Sites
+  - action: scrapeXPath
+    scraper: vrSceneScraper
+    url:
+      - lethalhardcorevr.com/
+      - whorecraftvr.com/
+
+movieByURL:
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      # - bangmytransass.com/ # Scenes published as movies?
+      # - blacktranswhackers.com/ # Scenes published as movies?
+      - concoxxxion.com/
+      - elegantangel.com/
+      - manipulativemedia.com/
+      - severesexfilms.com/
+      # - shemalestrokers.com/ # Scenes published as movies?
+      - thirdworldxxx.com/
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="description"]/text()
+      Date:
+        selector: //div[@class="release-date"][1]/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+      Details: //div[@class="synopsis"]//text()
+      Tags:
+        Name: //div[@class="tags"]/a/text()
+      Performers:
+        Name: //div[@class="video-performer"]/a//text()
+      Studio:
+        Name:
+          selector: //div[@class="studio"]/span[2]
+          postProcess:
+            # Override studio name only when needed
+            - map:
+                Jay Rock Clips: Jay's POV
+            - replace:
+                # 'Jane Doe Clips' => 'Jane Doe'
+                - regex: \sClips$
+                  with:
+      Image: //link[@rel="image_src"]/@href
+      Movies:
+        Name: //h1[@class="description"]/following-sibling::p/a/text()
+
+  movieScraper:
+    movie:
+      Name: //h1[@class="description"]/text()
+      Date:
+        selector: //div[@class="release-date"][1]/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+      Synopsis: //div[@class="synopsis"]//text()
```

Comment:
this needs either a concat to get all elements or a `//div[@class="synopsis"]` to get all the div
it might be needed for the scene scraper details selector also just to be safe  (i didnt encounter cases for scenes , i didnt try a lot though)

Reply:
Good idea, I will concat the `text()`s on both.